.github/workflows/unit-test.yaml

```diff
@@ -0,0 +1,35 @@
+name: Unit Test
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "release-**"
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+      - CODEOWNERS
+      - LICENSE
+
+jobs:
+  unit-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.12'
```

Comment:
the same point with another PR regarding extracting python version from toml file

Reply:
done!